docs/2.0/integration/junit5.adoc

```diff
@@ -0,0 +1,133 @@
+= jUnit 5
+
+== Installation
+
+The latest available version of `allure-junit5`:
+
+image::https://img.shields.io/maven-central/v/io.qameta.allure/allure-junit5.svg[Allure JUnit5]
+
+=== Maven
+
+Add the following to your *pom.xml*:
+[source, xml, linenums]
+.pom.xml
+----
+<properties>
+        <allure-junit5.version>2.0-BETA19</allure-junit5.version>
+        <aspectj.version>1.8.10</aspectj.version>
+        <!-- Please note that according to Junit 5 documentation, it will only work with maven surefire-plugin 2.19.1 at this time -->
```

Comment:
I rather prefer remove all other properties, and add this note as a note in doc

Reply:
removed all except aspectj.version, as it also appears in junit4